src/sasctl/_services/folders.py

```diff
@@ -135,3 +137,46 @@ def get_folder(cls, folder, refresh=False):
             return cls.get(f"/folders/{folder}")
 
         return cls._get_folder(folder, refresh=refresh)
+
+    @classmethod
+    def create_path(cls, folder, description=None):
+        """Create a new folder recursively.
+
+        Parameters
+        ----------
+        folder : str
+            The folder to be created including the path.
+        description: str, optional
+             A description of the folder
+
+        Returns
+        -------
+        RestObj
+            Details of newly-created folder
+
+        """
+        if isinstance(folder, str) and folder.count("/") >= 1:
```

Comment:
Rather than erroring if not passing a string, we can probably just use duck typing and do something like:
```
folder = str(folder)
if not folder.startswith("/"):
    folder = f"/{folder}"
```

Reply:
Done